pkg/utils/selector.go

```diff
@@ -106,10 +109,17 @@ func SelectPods(ctx context.Context, c client.Client, selector v1alpha1.Selector
 
 	pods = append(pods, podList.Items...)
 
-	namespaceSelector, err := parseSelector(strings.Join(selector.Namespaces, ","))
+	namespaceSelector, err := parseSelector(objMeta.GetNamespace())
```

Comment:
It seems this change would only let namespaceSelector using the chaos object own namespace instead of the selected namespaces?

Reply:
In fact, the full code for namespaceSelector is shown below
```golang
	namespaceSelector, err := parseSelector(objMeta.GetNamespace())
	if err != nil {
		return nil, err
	}
	if len(selector.Namespaces) != 0 {
		namespaceSelector, err = parseSelector(strings.Join(selector.Namespaces, ","))
		if err != nil {
			return nil, err
		}
	}
```
[code](https://github.com/Gallardot/chaos-mesh/blob/b320467cc44f8f61d1774e8ab4f5a8e7052be94d/pkg/utils/selector.go#L112-L121)

1. using the chaos object own namespace as default.
2. If `selector.Namespaces` is set, instead of the default.